java/server/test/org/openqa/grid/internal/ProxySetTest.java

```diff
@@ -58,77 +53,4 @@ public void removeIfPresent() {
       registry.stop();
     }
   }
-
-  @Test
-  public void testProxySortingByIdle() throws Exception {
```

Comment:
hmmm, well there was a test which in fact didn't test what it was supposed to because it didn't work (much respect to my friend dima). so in fact the test was worse than having a test as it made you think it worked but it didn't, as you can see from the issue!